resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStep.scala

```diff
@@ -70,7 +70,9 @@ private[spark] class BasicDriverFeatureStep(conf: KubernetesDriverConf)
   private val driverMemoryWithOverheadMiB = driverMemoryMiB + memoryOverheadMiB
 
   override def configurePod(pod: SparkPod): SparkPod = {
-    val driverCustomEnvs = conf.environment.toSeq
+    val driverEnvs = (Seq(
```

Comment:
Shall we keep the existing name `driverCustomEnvs` if there is no reason?

Reply:
https://github.com/apache/spark/blob/ddf83159a8c61fa12237a60124f7a7aa4e3a53c1/resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/features/BasicExecutorFeatureStep.scala#L102

maybe it's better to have consistent naming convention?

Reply:
If that is required, we can do that later in another PR.

Reply:
Sounds good. I change to `driverCustomEnvs`